Solve issue with scrollbars not being updated in tabbed browsing.

https://bugs.gentoo.org/show_bug.cgi?id=260965
https://bugzilla.gnome.org/show_bug.cgi?id=542396

Index: src/nautilus-navigation-window.c
===================================================================
--- src/nautilus-navigation-window.c	(revision 14834)
+++ src/nautilus-navigation-window.c	(working copy)
@@ -200,6 +200,12 @@
 	g_assert (slot != NULL);
 
 	nautilus_window_set_active_slot (nautilus_window, slot);
+	
+	if (slot->content_view != NULL)
+	{
+		gtk_scrolled_window_set_hadjustment (GTK_SCROLLED_WINDOW (slot->content_view), NULL);
+		gtk_scrolled_window_set_vadjustment (GTK_SCROLLED_WINDOW (slot->content_view), NULL);
+	}
 
 	return FALSE;
 }
